library/std/src/os/windows/process.rs

```diff
@@ -192,6 +192,20 @@ pub trait CommandExt: Sealed {
     /// ```
     #[unstable(feature = "windows_process_extensions_async_pipes", issue = "98289")]
     fn async_pipes(&mut self, always_async: bool) -> &mut process::Command;
+
+    /// Attach the specified proc thread attribute. Multiple attributes may be attached.
+    ///
+    /// # Safety
+    ///
+    /// - The attribute and value pair must be supplied in accordance with [Win32 API usage][1].
+    ///
+    /// [1]: https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-updateprocthreadattribute
```

Comment:
It would be good to have a minimal example here. Also, under safety, it'd be worth mentioning that any resources passed as attributes (file handles, pointers to memory, etc) must live at least as long as the `Command` exists.